NEWS.md

```diff
@@ -1,3 +1,41 @@
+### v8.11.0 (2022-05-11)
+
+* Log forwarding from APM agent is now possible.
+
+    * Added application log aggregator to collect logs with adaptive sampling and appropriate max samples stored.
+
+    * Added `application_logging` configuration and env vars with appropriate defaults.
+
+    * Changed default value of `application_logging.enabled` to be true.
```

Comment:
this is a little weird here.  because we haven't released application_logging as a config yet so how could it change before release?

Reply:
Agree, either strike this or change to "Set default value..."